File: College-Sports-WebApp/ClientApp/src/api/models/scoreboard-result.ts
/* tslint:disable */
/* eslint-disable */
import { Event } from '../models/event';
import { EventsDate } from '../models/events-date';
import { League } from '../models/league';
import { Video } from '../models/video';
export interface ScoreboardResult {
  events?: Array<Event> | null;
  eventsDate?: EventsDate;
  groups?: Array<string> | null;
  id?: number;
  leagues?: Array<League> | null;
  videos?: Array<Video> | null;
}

File: College-Sports-Domain/DTO/ScoreboardResult.cs
namespace College_Sports_WebApp.Database.Models
{
    public class ScoreboardResult
    {
        public int Id { get; set; }
        public List<League> leagues { get; set; }
        public List<Video> videos { get; set; }
        public List<string> groups { get; set; }
        public List<Event> events { get; set; }
        public EventsDate eventsDate { get; set; }
    }

}